src/plugins/intel_gpu/src/graph/convolution.cpp

```diff
@@ -341,20 +347,14 @@ convolution_inst::typed_primitive_inst(network& network, convolution_node const&
                               "Weights number of dimensions",
                               filter_inst.size.raw.size(),
                               "output number of dimensions",
-                              output_inst.size.raw.size(),
+                              output_layout.size.raw.size(),
                               "Weights/output dims mismatch");
         CLDNN_ERROR_NOT_EQUAL(node.id(),
                               "Convolution padding mode",
                               node.get_output_layout().data_padding.filling_value(),
                               "padding value",
                               0.0f,
                               "Unknown padding mode.");
-        CLDNN_ERROR_NOT_EQUAL(node.id(),
-                              "Pad number of dimensions",
-                              pad.raw.size(),
-                              "input number of dimensions",
-                              input_inst.size.raw.size(),
-                              "Pad/ input size mismatch");
```

Comment:
How about comparing `pad.size()` with `input_inst.get_output_layout().get_spatial_rank()` here as well?

Reply:
they can be different in some cases, e.g. for 1d convolution